biz.aQute.resolve/src/biz/aQute/resolve/AbstractResolveContext.java

```diff
@@ -105,6 +105,8 @@
 	private Set<Resource>							blacklistedResources		= new HashSet<Resource>();
 	private int										level						= 0;
 	private Resource								framework;
+	private Boolean									collectAllMissingRequirements;
```

Comment:
it allows for an external agent to know if the setting was previously set or not using Boolean as trinary state.